source/server/config/http/grpc_web.cc

```diff
@@ -0,0 +1,30 @@
+#include "server/config/http/grpc_web.h"
+
+#include "common/grpc/grpc_web_filter.h"
+
+namespace Envoy {
+namespace Server {
+namespace Configuration {
+
+HttpFilterFactoryCb GrpcWebFilterConfig::tryCreateFilterFactory(HttpFilterType type,
+                                                                const std::string& name,
+                                                                const Json::Object&,
+                                                                const std::string&,
+                                                                Server::Instance&) {
+  if (type != HttpFilterType::Both || name != "grpc_web") {
```

Comment:
just a heads up: https://github.com/lyft/envoy/pull/994
HTTP filters registration will be changed in ^